docs/blog/2020-06-16-Gatsby-slideshow-with-Posenet/index.md

```diff
@@ -438,7 +429,7 @@ When both wrists were in the same box, I knew I was gesturing for a slide change
 
 All that was left was to work out whether my poses array containing the positions of the left and right wrist had placed them both in the bounding box. I could then use my socket connection as above to change the slide.
 
-```js
+```javascript
```

Comment:
i would like to have here a file name, because i don't know where it belongs to (i can not find this code snipped in the linked server and client repos)

````suggestion
```javascript:title=???
````

Reply:
See comment above :)